app/controllers/weets.js

```diff
@@ -13,3 +14,15 @@ exports.createWeet = async (req, res, next) => {
     next(error);
   }
 };
+exports.getWeets = async (req, res, next) => {
+  try {
+    const allWeets = await weetService.getWeets();
+    const pagination = req.query;
```

Comment:
when you are getting variables from `req.query` or `req.body` try to put them up at the beginning.

```suggestion
    const pagination = req.query;
    const allWeets = await weetService.getWeets();
```